DESIGN.md

```diff
@@ -0,0 +1,108 @@
+# Vanagon
+- Note that this is an incomplete doc, documentation on all aspects of vanagon to come
+
+## Vanagon for the Windows platform
+
+### Overview
+Vanagon was adapted for the windows platform using WiX. At a very high level each component of the project gets built on a build system, then gets moved to a staging directory where we use the WiX toolset to build an MSI. This workflow is consistent with the rest of vanagon, however the WiX toolset is not so simple that there are no caveats, specifically:
```

Comment:
Put in URL links for Wix like you have in #375 
Also shouldn't that be "there are caveats"